src/main/resources/static/js/app.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Add jQuery Validation plugin method for a valid password
+ *
+ * Valid passwords contain at least one letter and one number.
+ */
+$.validator.addMethod(
+  "validPassword",
+  function (value, element, param) {
```

Comment:
we can handle it via Spring, but depends on you.This is only frontend validation that can be easily overwritten

Reply:
I would like also to try it with Spring, so I can remove this part of code.